### DISCUSSION TITLE
Keep original ESP and add extra ESP for local monitoring

### DISCUSSION THREAD

**Discussion by Quattrohead** [emporia-vue-local/esphome#93] [Ideas]
As per the title, can we keep the original ESP intact and add this project ESP to the uart pins on the Emporia.
I do like their system but also want real time monitoring for HA etc.

**Comment by flaviut** [comment]
Unfortunately, no.

Communication between the original ESP and the microcontroller doing the measurement is I2C, which is quite a bit harder to tap than UART. Also keep in mind a mod like this will likely mean your device is no longer UL listed.
